cardano-cli/src/Cardano/CLI/Types/Key/AnyVerificationKeySource.hs

```diff
@@ -0,0 +1,16 @@
+{-# LANGUAGE DataKinds #-}
+
+module Cardano.CLI.Types.Key.AnyVerificationKeySource
+  ( AnyVerificationKeySource(..)
+  ) where
+
+import           Cardano.Api
+
+import           Cardano.CLI.Types.Key.AnyVerificationKeyText
+
+-- | The source from which a verification key of an unspecified key role can be
+-- derivied.
+data AnyVerificationKeySource
```

Comment:
Using the naming convention that `XyzSource` is a thing from which an `Xyz` can be derived.

In this case, the `Xyz` is `VerificationKey`.  The `Any` prefix means any `keyrole`.